coalib/coala_main.py

```diff
@@ -100,34 +100,35 @@ def run_coala(console_printer=None,
     """
     all_actions_possible = provide_all_actions()
     apply_single = None
+
     if getattr(args, 'single_action', None) is not None:
         while True:
             for i, action in enumerate(all_actions_possible, 1):
                 console_printer.print(format_lines('{}'.format(
-                    action), symbol='['))
+                    action), symbol='[', line_nr=' ' + str(i) + '. '))
 
             line = format_lines(STR_ENTER_NUMBER, symbol='[')
-
             choice = input(line)
+            choice = str(choice)
 
-            if choice.isalpha():
+            if choice.isalnum():
                 choice = choice.upper()
                 choice = '(' + choice + ')'
```

Comment:
better to change this to `choice = '(' + choice + ')' if choice.isalpha()` will save you the trouble for comparing of numeric characters later